main.go

```diff
@@ -2,35 +2,73 @@ package main
 
 import (
 	"encoding/json"
+	"html/template"
 	"log"
 	"net/http"
 	"os"
+	"strings"
 
 	"github.com/Clever/mesos-visualizer/ecs"
 )
 
 var (
-	Cluster            string
+	Clusters           map[string]string
 	AWSAccessKeyID     string
 	AWSSecretAccessKey string
 )
 
 func init() {
-	Cluster = getEnv("CLUSTER")
+	Clusters = getEnvJSON("CLUSTERS")
 	AWSAccessKeyID = getEnv("AWS_ACCESS_KEY_ID")
 	AWSSecretAccessKey = getEnv("AWS_SECRET_ACCESS_KEY")
 
 }
 
 func main() {
-	http.HandleFunc("/resources.json", resourcesHandler)
-	http.Handle("/", http.StripPrefix("/", http.FileServer(http.Dir("./static"))))
+	http.HandleFunc("/resources/", resourcesHandler)
+	http.HandleFunc("/", indexHandler)
+	http.Handle("/static/", http.StripPrefix("/static/", http.FileServer(http.Dir("./static"))))
 	log.Print("Listening on port 80...")
 	log.Fatal(http.ListenAndServe(":80", nil))
 }
 
-func resourcesHandler(w http.ResponseWriter, r *http.Request) {
-	c := ecs.NewClient(Cluster, AWSAccessKeyID, AWSSecretAccessKey)
+func indexHandler(w http.ResponseWriter, r *http.Request) {
+	tmpl, err := template.New("home").Parse(`
+	<!DOCTYPE html>
+	<head>
+		<meta charset="utf-8">
+		<title>ECS Visualizations</title>
+	</head>
+	<body>
+		{{ range $name, $arn := . }}
```

Comment:
Do you need to specify the `$arn` name here? Or could you `_` it?

Reply:
It doesn't seem like go templates support the `_` syntax.